Tools/CISupport/ews-build/steps.py

```diff
@@ -263,7 +263,7 @@ def get_number_of_prs_with_label(self, label):
         query = {'query': query_body}
         response = yield self.query_graph_ql(query)
         if response:
-            num_prs = response['data']['repository']['pullRequests']['totalCount']
+            num_prs = (((response.get('data') or {}).get('repository') or {}).get('pullRequests') or {}).get('totalCount') or 0
```

Comment:
Any of these fields being null would imply not retrieving the actual results, so there could be n PRs in safe-merge-queue but it would look like there's 0. I think any of these being null should result in an error instead of defaulting to 0.